Source/WebCore/css/parser/CSSParserImpl.h

```diff
@@ -180,14 +180,30 @@ class CSSParserImpl {
         ASSERT(!m_nestingContextStack.isEmpty());
         return m_nestingContextStack.last();
     }
-    bool isNestedContext()
+
+    bool isStyleNestedContext()
     {
         return (m_isAlwaysNestedContext == CSSParserEnum::IsNestedContext::Yes || m_styleRuleNestingLevel) && context().cssNestingEnabled;
     }
 
+    bool isNestedContext()
+    {
+        auto isScopeNested = m_scopeRuleNestingLevel && context().cssScopeAtRuleEnabled;
```

Comment:
I don't think m_scopeRuleNestingLevel is ever incremented if cssScopeAtRuleEnabled is not set so testing for the latter is unnecessary.